packages/babel-preset-expo/src/environment-restricted-imports.ts

```diff
@@ -14,7 +14,7 @@ export function environmentRestrictedImportsPlugin(
 ): babel.PluginObj {
   const { types: t } = api;
 
-  const isReactServer = api.caller(getIsReactServer);
+  const isReactServer = api.caller(getIsReactServer) || api.caller(getIsServer);
```

Comment:
What is the difference between "react server" and "server"?

Reply:
isServer is traditional server rendering (used for API routes and SSG) whereas the new React Server environment is used for RSC and Server Actions. The primary difference is the resolution patterns (`react-server` in package.json) and some of the additional rules such as "use client" and "use server" support.

Reply:
OK, I would rename this then, assuming that the React Server environment is a more precise environment of the traditional server environment:
```suggestion
  const isServer = api.caller(getIsReactServer) || api.caller(getIsServer);
```

Or if the traditional server environment isn't strictly more general than the React Server environment, we might want:

```
              ┌────────┐             
       ┌──────┤ server ├──────┐      
       │      └────────┘      │      
       │                      │      
       │                      │      
┌──────▼───────┐       ┌──────▼─────┐
│ react server │       │ api server │
└──────────────┘       └────────────┘
```
```suggestion
  const isServer = api.caller(getIsReactServer) || api.caller(getIsAPIServer);
```

Reply:
sounds good